include/dlaf/eigensolver/eigensolver.h

```diff
@@ -11,8 +11,6 @@
 
 /// @file
 
-#include <utility>
-
```

Comment:
inshpect rightfully complains about this header being removed. It's required for `std::move` (and possibly more).

Reply:
Yep, thanks. I need to prevent the tooling I have to change the source, it’s proving quite un-reliable.